index.js

```diff
@@ -0,0 +1,3 @@
+'use strict',
```

Comment:
Missing semicolon.<br>Expected an identifier and instead saw ','.<br>Expected an assignment or function call and instead saw an expression.